index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Porfolio Website</title>
+    <title>Portfolio Website</title>
```

Comment:
- [ ] When I hover over the buttons the cursor does not change to pointer 👆🏼. Please fix this issue by adding them `cursor: pointer` on hover 🍀🙌🏼

Reply:
@Rustamxon7 Thank you for reviewing my project, I will make the necessary corrections as suggested